package.json

```diff
@@ -78,18 +78,18 @@
     "collectCoverageFrom": [
       "src/**/*.ts"
     ],
-    "globals": {
-      "ts-jest": {
-        "babelConfig": false
-      }
-    },
     "moduleFileExtensions": [
       "ts",
       "js",
       "json"
     ],
     "transform": {
-      "^.+\\.ts$": "ts-jest"
+      "^.+\\.ts$": [
+        "ts-jest",
+        {
+          "babelConfig": false
+        }
+      ]
```

Comment:
You mentioned facing some issues when trying to update those dependencies last week. Was it related to this and Jest ?

Reply:
No, I tried to update **everything**, including TS. So there were many many issues. In this PR, I only targeted the required dependencies and a few others around them